### ISSUE TITLE
symbiont process

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-annotation#1891]
human has 1403 annotations to "symbiont process". I find these very misleading.
The practice of annotating human genes to viral processes seems very strange to me full stop...

**Comment by ValWood** [comment]
for example 600+  human 
genes

annotated to
Definition
A set of processes which all viruses follow to ensure survival; includes attachment and entry of the virus particle, decoding of genome information, translation of viral mRNA by host ribosomes, genome replication, and assembly and release of viral particles containing the genome. Source: ISBN:1555811272

...makes it very difficult to identify/ analyse known/unknown human processes, these don't fit the proposed definition of a GO process

**Comment by ValWood** [comment]
"A biological process represents a specific objective that the organism is genetically “programmed” to achieve.
https://github.com/geneontology/go-ontology/issues/14899

**Comment by pgaudet** [comment]
We need to fix the ontology (there are lots of tickets). There are many children that are either dubious or at the wrong place. Examples: 
- autophagy of host cells involved in interaction with symbiont (not sure what that means - is this a symbiont or a host process?)
- regulation of entry of bacterium into host cell (dubious)
- negative regulation by host of viral genome replication (not a symbiont process)

- etc ! 

Pascale

**Comment by ValWood** [comment]
OK, we'll ignore these terms for our purposes...

**Comment by ValWood** [comment]
you can close this if it's covered...

**Comment by cmungall** [comment]
Relevant ticket:
https://github.com/geneontology/go-ontology/issues/14807

I think many of the part-of children of symbiont process should in fact be regulates.

**Comment by ValWood** [comment]
> I think many of the part-of children of symbiont process should in fact be regulates.

I don't think that solves the problem.

Human genes are not "regulating" viral processes....virus are co-opting human processes for their own purposes..... A big difference.

**Comment by cmungall** [comment]
Ah sorry I realized I forgot to give context.

I just picked a random term with annotations under 'symbiont process', http://amigo.geneontology.org/amigo/term/GO:0070946, neutrophil mediated killing of gram-positive bacterium.  (a bacterial one, not viral)

![symb](https://user-images.githubusercontent.com/50745/38058233-04655092-3297-11e8-8b72-9fb17f902d3c.png)


There are a couple of mouse genes with experimental annotations to this. So this was bacterial and there isn't any co-option involved. So it's hopefully more straightforward, but in fact there are a number of issues here.

First is this:

```
id: GO:0051702
name: interaction with symbiont
relationship: part_of GO:0044403 ! symbiont process
```

This doesn't seem right but I'm not sure I fully understand the new definition of symbiont process (which is kind of an odd term no biologist would use, the only hits in google are to GO, or apparently "symbiont process engineer" is a job at hewlett packard). At least with a traditional definition of symbiosis, it's not the case that all interactions with the symbiont would be symbiotic interactions. For example, if my gene products enable some defense against a parasite, that isn't part of the general parasitism process. But it could be considered regulating the activities that are part of the parasitism.

but there is also a problem here:

```
id: GO:0070944
name: neutrophil mediated killing of bacterium
is_a: GO:0070943 ! neutrophil mediated killing of symbiont cell
...
```

AFAIK it's only a symbiont if there is some kind of long-term interaction. Is it considered a parasite if it's a short term opportunistic infection? I suppose arguments can be made that all bacteria on a host can be regarded somewhere on the spectrum commensal/mutual/parasite regardless of duration of interaction. But this then renders the concept of symbiont to be so broad there is little point having a distinction between symbiont process and the parent 'interspecies interaction between organisms', the only differentia is one of relative size.
